nixos/doc/manual/configuration/gpu-accel.xml

```diff
@@ -100,6 +100,35 @@ ROCR_EXT_DIR=`nix-build '&lt;nixpkgs&gt;' --no-out-link -A rocm-runtime-ext`/lib
   Image support      Yes</screen>
       </para>
     </section>
+
+    <section xml:id="sec-gpu-accel-opencl-intel">
+      <title>Intel</title>
+
+      <para>
+       <link
+         xlink:href="https://en.wikipedia.org/wiki/List_of_Intel_graphics_processing_units#Gen8">Intel
+       Gen8 and later GPUs</link> are supported by the Intel NEO OpenCL
+       runtime that is provided by the
+       <package>intel-compute-runtime</package> package. For Gen7 GPUs,
+       the deprecated Beignet runtime can be used, which is provided
+       by the <package>beignet</package> package. The proprietary Intel
+       OpenCL runtime, in the <package>intel-ocl</package> package, is
+       an alternative for Gen7 GPUs.
```

Comment:
Also an alternative  for Gen8 apparently.

Reply:
I am wondering, rereading your clinfo output, if that isn’t actually their CPU implementation. The device name is the CPU and not the GPU, plus the listed device type is CPU. Their doc states supporting only 7th gen GPUs:

https://software.intel.com/content/www/us/en/develop/tools/opencl-sdk.html

(If this is the right documentation, the Intel site is confusing...)

Reply:
Oh ok, so I'll again take a closer look. Since the GPU is integrated into the CPU I didn't really wonder that it showed the CPU.